lib/workerholic/statistics_storage.rb

```diff
@@ -11,7 +11,7 @@ def self.save_job(category, job)
     private
 
     def self.storage
-      storage ||= Storage::RedisWrapper.new
+      Storage::RedisWrapper.new
```

Comment:
you should use a class instance variable @storage so you don't keep creating instances of `Storage::RedisWrapper.new`